lib/models/index.js

```diff
@@ -13,10 +13,11 @@ var dbconfig = config.db;
 dbconfig.logging = config.debug ? logger.info : false;
 
 var sequelize = null;
-if (dbconfig.hasOwnProperty('username') || dbconfig.hasOwnProperty('password'))
-    sequelize = new Sequelize(dbconfig.database, dbconfig.username, dbconfig.password, dbconfig);
+
+if (process.env.DATABASE_URL)
+    sequelize = new Sequelize(process.env.DATABASE_URL, dbconfig.username, dbconfig.password, dbconfig);
```

Comment:
@SISheogorath This fix is borrowed from the official sequelize example https://github.com/sequelize/express-example/blob/master/models/index.js
And it's ok to pass undefined parameter to the constructor according to their document http://sequelize.readthedocs.io/en/latest/api/sequelize/

I believe this will remain backwards compatibility because usually you'll fall into the full constructor.